content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -47,18 +47,45 @@ from within your Istio cluster. This task shows you how to access an external HT
     EOF
     {{< /text >}}
 
-1.  Exec into the `sleep service` source pod:
+1.  Make a request to the external HTTP service from `SOURCE_POD`:
 
     {{< text bash >}}
-    $ kubectl exec -it $SOURCE_POD -c sleep sh
+    $  kubectl exec -it $SOURCE_POD -c sleep -- curl http://httpbin.org/headers
+    {
+      "headers": {
+      "Accept": "*/*",
+      "Connection": "close",
+      "Host": "httpbin.org",
+      "User-Agent": "curl/7.60.0",
+      ...
+      "X-Envoy-Decorator-Operation": "httpbin.org:80/*",
+      "X-Istio-Attributes": ...
+      }
+    }
     {{< /text >}}
 
-1.  Make a request to the external HTTP service:
+    Note the headers added by the Istio sidecar proxy: `X-Envoy-Decorator-Operation` and `X-Istio-Attributes`.
+
+1.  Check the log of the sidecar proxy of `SOURCE_POD`:
 
     {{< text bash >}}
-    $ curl http://httpbin.org/headers
+    $  kubectl logs $SOURCE_POD -c istio-proxy | tail
+    [2019-01-24T12:17:11.640Z] "GET /headers HTTP/1.1" 200 - 0 599 214 214 "-" "curl/7.60.0" "17fde8f7-fa62-9b39-8999-302324e6def2" "httpbin.org" "35.173.6.94:80" outbound|80||httpbin.org - 35.173.6.94:80 172.30.109.82:55314 -
     {{< /text >}}
 
+    Note the entry related to your HTTP request to `httpbin.org/headers`.
+
+1.  Check the Mixer log. If Istio is deployed in the `istio-system` namespace, the command to print the log is:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system logs -l istio-mixer-type=telemetry -c mixer | grep 'httpbin.org'
+    {"level":"info","time":"2019-01-24T12:17:11.855496Z","instance":"accesslog.logentry.istio-system","apiClaims":"","apiKey":"","clientTraceId":"","connection_security_policy":"unknown","destinationApp":"","destinationIp":"I60GXg==","destinationName":"unknown","destinationNamespace":"default","destinationOwner":"unknown","destinationPrincipal":"","destinationServiceHost":"httpbin.org","destinationWorkload":"unknown","grpcMessage":"","grpcStatus":"","httpAuthority":"httpbin.org","latency":"214.661667ms","method":"GET","permissiveResponseCode":"none","permissiveResponsePolicyID":"none","protocol":"http","receivedBytes":270,"referer":"","reporter":"source","requestId":"17fde8f7-fa62-9b39-8999-302324e6def2","requestSize":0,"requestedServerName":"","responseCode":200,"responseSize":599,"responseTimestamp":"2019-01-24T12:17:11.855521Z","sentBytes":806,"sourceApp":"sleep","sourceIp":"AAAAAAAAAAAAAP//rB5tUg==","sourceName":"sleep-88ddbcfdd-rgk77","sourceNamespace":"default","sourceOwner":"kubernetes://apis/apps/v1/namespaces/default/deployments/sleep","sourcePrincipal":"","sourceWorkload":"sleep","url":"/headers","userAgent":"curl/7.60.0","xForwardedFor":"0.0.0.0"}
+    {{< /text >}}
+
+    Note that the `destinationServiceHost` attribute is equal to `httpbin.org`. Also note HTTP-related attributes:
```

Comment:
Also notice the HTTP-related ...

Reply:
Fixed in https://github.com/istio/istio.io/pull/3222/commits/5b9ad33f13339d91904908cbab142cb4deb73197